app/robots/iCubGenova02/hardware/motorControl/left_upper_leg-ems6-mc.xml

```diff
@@ -11,7 +11,7 @@
 
     <group name="LIMITS">
         <param name="jntPosMax">                      85            85            70            0     </param>
-        <param name="jntPosMin">                     -30             0           -70         -100     </param>
```

Comment:
It is good practice to avoid to commit changes not  directly related to what you want to change.